apps/code-examples/src/app/code-examples/lookup/lookup/async/lookup-async-demo.component.ts

```diff
@@ -53,17 +52,6 @@ export class LookupAsyncDemoComponent implements OnInit {
     });
   }
 
-  // Only show people in the search results that have not been chosen already.
-  public getSearchFilters(): SkyAutocompleteSearchFunctionFilter[] {
```

Comment:
Why are we removing `getSearchFilters`?

Reply:
Search filters aren't used in async search